apps/chat/environment.d.ts

```diff
@@ -11,6 +11,9 @@ declare global {
       APP_BASE_ORIGIN?: string;
       ALLOWED_IFRAME_ORIGINS?: string;
       IS_IFRAME?: string;
+      ALLOWED_IFRAME_SOURCES?: string;
+      CUSTOM_VISUALIZERS?: string;
+      CUSTOM_VISUALIZERS_TYPES?: string;
```

Comment:
These new variables should be mentioned in chat README with links to `libs/visualizer-connector/README.md`
Also chat README should have section/link for custom renderers as we hae for overlay